app/src/main/java/cloudcity/networking/models/MeasurementsModel.java

```diff
@@ -22,6 +22,8 @@ public class MeasurementsModel extends NetworkDataModel.NetworkDataModelValues {
     @SerializedName("dummy_value")
     private Integer dummy;
 
+    private transient String cellType;
```

Comment:
no, it's transient because we don't want to send it with the typical sending model; the other subclasses of `NetworkDataModel` have their own categories; this one is just so the Iperf3NetworkDataModel could fill in one value in it's Iperf3ValuesModel member/class.

won't change.

Reply:
resolving.